plugins/catalog-backend/src/ingestion/processors/index.ts

```diff
@@ -35,3 +35,6 @@ export * from './types';
 export { UrlReaderProcessor } from './UrlReaderProcessor';
 export { parseEntityYaml } from './util/parse';
 export { results };
+
+export { BitbucketClient } from './bitbucket';
```

Comment:
Is it possible to export a slimmer interface for this instead? Exporting the entire client is a bit of an API surface, don't want people to be depending on it tbh 😅. It's only used when passed to the parser function right?

Reply:
Yes, it is only used when it is passed to the parser, to make it easier to ask for other files. An alternative is to not pass it at all, but instead pass the `ScmIntegration` and then let the implementor handle further calls to the Bitbucket API however he/she think is appropriate. That way we do not need to expose the bitbucket-client at all.

Reply:
I have updated the pull request to pass the `BitbucketIntegration` object instead of the internal `BitbucketClient`